src/osd/ECBackend.cc

```diff
@@ -1642,11 +1846,15 @@ struct CallClientContexts :
 	   ++j) {
 	to_decode[j->first.shard].claim(j->second);
       }
-      ECUtil::decode(
+      int r = ECUtil::decode(
```

Comment:
probably, we can leave as it is or simply `assert(r == 0)`. as decode() never fails. it does not check for inconsistency. the only reason that it fails is that we fail to offer enough chunks. and it is not possible in our case.